### PR TITLE
simpler fetch/refetch

### PR DESCRIPTION
**Pull request by omortie**

### CODE REVIEW

**Comment by omortie** [comment]
related to #433

**Comment by omortie** [comment]
@nrbnlulu thanks, me too, I expected tests to run, but they seem to have been disabled somehow, I can also help

File: src/qtgql/gqltransportws/detail/gqltransportws.cpp [hunk 1]
```diff
@@ -201,17 +209,27 @@ bool GqlTransportWs::gql_is_valid() const {
   return is_valid() && m_connection_ack;
 }
 
-void GqlTransportWs::execute(
-    const std::shared_ptr<bases::HandlerABC> &handler) {
-  auto new_op_id = QUuid::createUuid();
-  execute_impl(new_op_id, handler);
```
Comment (nrbnlulu) [inline src/qtgql/gqltransportws/detail/gqltransportws.cpp:207]: I think `execute_impl` is redundant by now am I wrong?

File: src/qtgql/routers/detail/routers.cpp [hunk 2]
```diff
@@ -6,11 +6,11 @@ qtgql::routers::SubscriptionRouter::SubscriptionRouter(
...
 void qtgql::routers::SubscriptionRouter::execute(
-    const std::shared_ptr<bases::HandlerABC> &handler) {
+    const std::shared_ptr<bases::HandlerABC> &handler, QUuid op_id) {
   auto message = handler->message();
   if (message.query.startsWith("subscription")) {
-    ws_layer->execute(handler);
+    ws_layer->execute(handler, op_id);
   } else {
-    http_layer->execute(handler);
+    http_layer->execute(handler, op_id);
```
Comment (nrbnlulu) [inline src/qtgql/routers/detail/routers.cpp:14]:
```suggestion
    http_layer->execute(handler, op_id);
    
```

Comment (nrbnlulu) [reply src/qtgql/routers/detail/routers.cpp:14]: just to see if it would trigger the tests

**Comment by nrbnlulu** [comment]
tests havent been triggered because you haven't modified anything in the tests (I should fix this) anyways it would fail since the tests are using the `refetch` function.

**Comment by nrbnlulu** [comment]
Closing since #477 was merged.

**Pull request closed without merging**